apps/src/p5lab/P5Lab.js

```diff
@@ -1495,11 +1495,6 @@ P5Lab.prototype.displayFeedback_ = function() {
     // add 'impressive':true to non-freeplay levels that we deem are relatively impressive (see #66990480)
     showingSharing:
       !level.disableSharing && level.freePlay /* || level.impressive */,
```

Comment:
comments regarding `impressive` can safely be removed from the comments on 2 lines here